stack.yaml.lock

```diff
@@ -0,0 +1,142 @@
+# This file was autogenerated by Stack.
```

Comment:
@mbj Should this file be included in git? Or should it be removed and a `gitignore` entry added for it?

Reply:
Yes, its the equivalent of a `Gemfile.lock`.